src/main/java/com/spotinst/sdkjava/SpotinstClient.java

```diff
@@ -115,6 +115,10 @@ public static SpotStorageAzureVolumeClient getVolumeClient(String authToken, Str
         SpotStorageAzureVolumeClient spotStorageAzureVolumeClient = new SpotStorageAzureVolumeClient(authToken, account);
         return spotStorageAzureVolumeClient;
     }
+    public static SpotOceanEcsClusterClient getOceanEcsClusterClient(String authToken, String account) {
```

Comment:
Should it be named SpotOceanEcsLaunchSpecClient?

Reply:
Same method will be used for cluster also